boot/boot_serial/src/boot_serial.c

```diff
@@ -125,7 +129,7 @@ static zcbor_state_t cbor_state[2];
 void reset_cbor_state(void)
 {
     zcbor_new_encode_state(cbor_state, 2, (uint8_t *)bs_obuf,
-        (size_t)bs_obuf + sizeof(bs_obuf), 0);
+        sizeof(bs_obuf), 0);
```

Comment:
Also when using default configs?

Reply:
yes - it looks like a general error (overflow not detected) that I found because I needed more space for the new decorate config. Will merge as a separate commit and create upstream issue with link.